### PR TITLE
Split namespace and path only on first colon, fixes #514

### LINKED ISSUE #514: @namespace error on Windows 7
I get error messages like the following when I try to include a template using a namespace:
TwigException: Unable to find template file C:\xampp\htdocs\apps\theme.test\web\themes\custom\themefolder\C/navigation/navigation.twig

UPDATE: Found the problem. In kss_builder_base_twig.js, at line 76, a namespace is split by a ":" delimiter. For Windows, if the path starts with "C:\" or another drive, then you end up with "namespace:C:\folder\subfolder" which would result in "C" as the path for the namespace.

I encountered this error while using [MediaCurrent's Theme Generator](https://github.com/mediacurrent/theme_generator_8). In one of the gulp tasks, kss namespace is set with __dirname, which is how "C:\" ended up as the path.

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

**Comment by DanielRuf** [comment]
I'll fix the codestyle today.

**Comment by DanielRuf** [comment]
Probably we should use `husky` to run the checks before we push / commit.

File: builder/base/twig/kss_builder_base_twig.js [hunk 1]
```diff
@@ -73,9 +73,10 @@ class KssBuilderBaseTwig extends KssBuilderBase {
       };
       this.options.namespace.forEach(namespace => {
         // namespace should be of the form "namespace:path";
-        let tokens = namespace.split(':', 2);
-        if (tokens[1]) {
-          this.namespaces[tokens[0]] = path.resolve(tokens[1]);
+        let namespacePart = namespace.substr(0, namespace.indexOf(':'));
```
Comment (ryuran) [inline builder/base/twig/kss_builder_base_twig.js]:
```suggestion
        const namespacePart = namespace.substr(0, namespace.indexOf(':'));
```

File: builder/base/twig/kss_builder_base_twig.js [hunk 2]
```diff
@@ -73,9 +73,10 @@ class KssBuilderBaseTwig extends KssBuilderBase {
...
+        let pathPart = namespace.substr(namespace.indexOf(':') + 1);
```
Comment (ryuran) [inline builder/base/twig/kss_builder_base_twig.js]:
```suggestion
        const pathPart = namespace.substr(namespace.indexOf(':') + 1);
```

File: builder/base/twig/kss_builder_base_twig.js (hunk 1 above)
Comment (DanielRuf) [reply builder/base/twig/kss_builder_base_twig.js]: I just changed the logic, it was already `let` ;-)

File: builder/base/twig/kss_builder_base_twig.js (hunk 2 above)
Comment (DanielRuf) [reply builder/base/twig/kss_builder_base_twig.js]: Same here. Generally there are not really constants but scoped variables.

File: builder/base/twig/kss_builder_base_twig.js (hunk 1 above)
Comment (ryuran) [reply builder/base/twig/kss_builder_base_twig.js]: I saw it ^^ old code is not the cleanest.

File: builder/base/twig/kss_builder_base_twig.js (hunk 2 above)
Comment (ryuran) [reply builder/base/twig/kss_builder_base_twig.js]: Yes, `const` are only scoped variable "unreasignable".

**Pull request merged**